metadata-service/factories/src/main/java/com/linkedin/metadata/boot/kafka/DataHubUpgradeKafkaListener.java

```diff
@@ -85,7 +85,7 @@ public void checkSystemVersion(final ConsumerRecord<String, GenericRecord> consu
       if (expectedVersion.equals(event.getVersion())) {
         IS_UPDATED.getAndSet(true);
       } else {
-        log.debug("System version is not up to date: {}", expectedVersion);
+        log.info("System version is not up to date: {}. Waiting for datahub-upgrade to complete...", expectedVersion);
```

Comment:
Are we worried at all that this might be noisy?

Reply:
not. its critical we add this